packages/core/src/lib/core/utils.ts

```diff
@@ -167,26 +166,6 @@ export function getDBFieldKeyForFieldOnMultiField(fieldKey: string, subField: st
   return `${fieldKey}_${subField}`;
 }
 
-// this whole thing exists because Prisma doesn't handle doing multiple writes on SQLite well
-// https://github.com/prisma/prisma/issues/2955
```

Comment:
Resolved https://github.com/prisma/prisma/issues/2955

Reply:
I had to drop this from this pull request, the change isn't a straight forward for the other databases